README.md

```diff
@@ -10,11 +10,16 @@ Installation
 
 To install, first clone the repository and then use pip to install:
 ```
-git clone git@github.com:sara-nl/picasclient.git
+git clone https://github.com/sara-nl/picasclient.git
 cd picasclient
 pip install -U .
 ```
 
+If you come across error messages related to python, you can execute pip module for a specific python version using the corresponding python:
+```
+python3.9 -m  pip install -U .
```

Comment:
I would suggest working in a virtual environment, but this will be addressed in this branch: https://github.com/sara-nl/picasclient/tree/convert-setuptools2poetry